src/test/java/com/wikia/webdriver/testcases/visualeditor/VEGalleryTests.java

```diff
@@ -140,9 +130,7 @@ public void VEGalleryTests_005_Remove() {
     ve.verifyEditorSurfacePresent();
     ve.deleteGallery(0);
     ve.verifyGalleries(0);
-    VisualEditorSaveChangesDialog saveChangesDialog = ve.clickPublishButton();
-    article = saveChangesDialog.savePage();
-    article.verifyVEPublishComplete();
+    ArticlePageObject article = ve.publish();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename "article" which hides the field declared at line 20. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AHiddenFieldCheck)